meta.go

```diff
@@ -266,19 +272,27 @@ func setMetaValueRecursive(key string, value string, previousMeta interface{}) (
 			var tmpValue interface{}
 			previousMetaMap := convertInterfaceToMap(previousMeta)
 			if previousMetaMap[keyHead] == nil {
-				childKey, tmpValue = setMetaValueRecursive(childKey, value, previousMetaMap)
+				childKey, tmpValue = setMetaValueRecursive(childKey, value, previousMetaMap, jsonValue)
 			} else {
 				// copy previous object only if it is map
 				previousObj := convertInterfaceToMap(previousMetaMap[keyHead])
 				if len(previousObj) != 0 {
 					obj = previousObj
 				}
-				childKey, tmpValue = setMetaValueRecursive(childKey, value, previousMetaMap[keyHead])
+				childKey, tmpValue = setMetaValueRecursive(childKey, value, previousMetaMap[keyHead], jsonValue)
 			}
 			obj[childKey] = tmpValue
 			return keyHead, obj
 		}
 	}
+	if jsonValue {
+		var objectValue interface{}
+		if err := json.Unmarshal([]byte(value), &objectValue); err == nil {
+			return key, objectValue
+		} else {
```

Comment:
don't need and shouldn't have `else` after return

Reply:
#23